clients/cli/src/copy.rs

```diff
@@ -8,10 +8,12 @@ use lockbook_core::{
 
```

Comment:
nitpick: Let's organize imports by grouping and in the following order:
1. std
2. third party libraries
3. lockbook_core
4. crate stuff